tests/elements-regression/src/controls/media.js

```diff
@@ -0,0 +1,63 @@
+const { ControlBase } = require( './control-base' );
+const media = require( '../media' );
+
+class Media extends ControlBase {
+	getSelector() {
+		return `.elementor-control-${ this.config.name }.elementor-control-type-media`;
+	}
+
+	async getValue() {
+		await this.openMediaLibrary();
```

Comment:
It will take time to sync the value from the `settings` model to the dom. it is possible but feels like it does not worth it. (we can create a new task about it)